x-pack/plugins/apm/server/lib/services/get_service_transaction_groups/get_transaction_groups_for_page.ts

```diff
@@ -122,7 +124,7 @@ export async function getTransactionGroupsForPage({
           latencyAggregationType,
           aggregation: bucket.latency,
         }),
-        throughput: bucket.transaction_count.value,
+        throughput: bucket.transaction_count.value / deltaAsMinutes,
```

Comment:
Same here: I'm surprised this wasn't caught by our API tests

Reply:
@dgieselaar This is a good example why value-based testing (and not just type based testing) is important.

Reply:
Snapshots won't catch this, as it was "new'. This would require API tests where you index documents where you know the TPM and then compare the output of the API with the TPM you'd expect.

Reply:
I see. I thought it also affected the inventory page and the transactions overview page. Seems like it is only the new service overview page being affected.